File: CatchTheKenny/app/src/main/java/com/mustafauysal/catchthekenny/MainActivity.java
package com.mustafauysal.catchthekenny;

import android.os.CountDownTimer;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.TextView;

import org.w3c.dom.Text;

public class MainActivity extends AppCompatActivity {

    TextView scoreText;
    TextView timeText;
    int score;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        //initialize

        timeText = (TextView) findViewById(R.id.timeText);
        scoreText = (TextView) findViewById(R.id.scoreText);

        score = 0;

        new CountDownTimer(10000,1000) {

            @Override
            public void onTick(long millisUntilFinished) {
                timeText.setText("Time: " + millisUntilFinished/1000);
            }

            @Override
            public void onFinish() {

            }
        }.start();
    }

    public void increaseScore (View view) {

        score++;
        //score  = score + 1;

        scoreText.setText("Score: " + score);

    }
}

File: StoringData/app/src/main/java/com/musta/storingdata/MainActivity.kt
package com.musta.storingdata

import android.content.Context
import android.content.SharedPreferences
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    lateinit var sharedPreferences : SharedPreferences

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)


        //SharedPreferences Initialize
        sharedPreferences = this.getSharedPreferences("com.musta.storingdata", Context.MODE_PRIVATE)

        val ageFromPreferences = sharedPreferences.getInt("age", -1)

        if (ageFromPreferences == -1) {
            textView.text = "Your Age: "
        } else {
            textView.text = "Your Age:  $ageFromPreferences"
        }
    }

    fun save (view : View) {

        //SharedPreferences


        val myAge = editText.text.toString().toIntOrNull()

        if (myAge != null) {
            textView.text = "Your Age: " + myAge
            sharedPreferences.edit().putInt("age", myAge).apply()
        }

    }

    fun delete (view : View) {

    }
}
